tests/unit/testErrorMessage.js

```diff
@@ -14,316 +14,67 @@
-    );
-
-    t.assert(
-      emp.hasOwnProperty('setFilePath')
-      , 'It should have a property named setFilePath'
-    );
-
-    t.assert(
-      ((typeof emp.setFilePath) === 'function')
-      , 'setFilePath should be of type function'
-    );
-
-    t.assert(
-      emp.hasOwnProperty('setLineNumber')
-      , 'It should have a property named setLineNumber'
-    );
-
-    t.assert(
-      ((typeof emp.setLineNumber) === 'function')
-      , 'setLineNumber should be of type function'
-    );
-
-    t.assert(
-      emp.hasOwnProperty('setFunctionName')
-      , 'It should have a property named setFunctionName'
-    );
-
-    t.assert(
-      ((typeof emp.setFunctionName) === 'function')
-      , 'setFunctionName should be of type function'
-    );
-
-    t.assert(
-      emp.hasOwnProperty('consumeRequestInformation')
-      , 'It should have a property named consumeRequestInformation'
-    );
-
-    t.assert(
-      ((typeof emp.consumeRequestInformation) === 'function')
-      , 'consumeRequestInformation should be of type function'
-    );
-  }
-)
-
-test(
-  'Instantiating a new ErrorMessage'
-  , function ( t ) {
-
-    var em = new ErrorMessage();
-
-    t.plan(19);
-
-    t.assert(
-      em.hasOwnProperty('eventTime')
-      , 'It should have a property named event time'
-    );
-
-    t.assert(
-      ((typeof em.eventTime) === 'string')
-      , 'The event time property should be of type String'
-    );
-
-    t.assert(
-      em.hasOwnProperty('serviceContext')
-      , 'It should have a service context property'
-    );
-
-    t.assert(
-      ((typeof em.serviceContext) === 'object')
-      , 'The service context property should be of type object'
-    );
-
-    t.deepEqual(
-      em.serviceContext
-      , { service: "node", version: undefined }
-      , [
-          'The service context property should have two properties:'
-          , 'service and version both with a string value of: default'
-        ].join(" ")
-    );
-
-    t.assert(
-      em.hasOwnProperty('message')
-      , 'It should have a property named message'
-    );
-
-    t.assert(
-      ((typeof em.message) === 'string')
-      , 'The message property should be of type string'
-    );
-
-    t.assert(
-      em.message === ""
-      , "The message property should be inited to an empty string"
-    );
-
-    t.assert(
-      em.hasOwnProperty('context')
-      , 'It should have a property named context'
-    );
-
-    t.assert(
-      ((typeof em.context) === 'object')
-      , 'The context property should be of type object'
-    );
-
-    t.assert(
-      em.context.hasOwnProperty('httpRequest')
-      , 'The context property should have a property named httpRequest'
-    );
-
-    t.assert(
-      ((typeof em.context.httpRequest) === 'object')
-      , "The context.httpRequest property should be of type object"
-    );
-
-    t.deepEqual(
-      em.context.httpRequest
-      , {
+describe('Instantiating a new ErrorMessage', function () {
```

Comment:
The reason they are split out is to clearly demonstrate the default values for each property. This means if a single property is changed in default only that test will break. I think this makes it more robust